roles/online_docs/templates/mkdocs/docs/archive.md

```diff
@@ -26,27 +26,27 @@ The following guidelines apply to this archive storage
 
 Archive is automatically mounted when user navigates to the `/groups/[GROUP]/arc[XX]` folder. At that moment storage from remote server gets mounted on the folder. It remains accessible until some specific idle time is reached.
 
-Data-manager account of the specific group is the **only** account that has **read** or **write** access to the archive folder of the group. This is to prevent the potential problems of accidentally recalling files online when not needed. Also to make sure that all the files are stored in correct format (see 'Best practices' below).
+Data-manager account of the specific group is the **only** account that has **read** and **write** access to the archive folder of the group. This is to prevent users accidentally recalling files online when not needed. Also to make sure that all the files are stored in correct format (see 'Best practices' below).
 
 ## 2. Managing data
 
-After some time, all the files on remote archive server get automatically migrated to the tape. When this happens, all the folders and files can be still normally seen in the structure. The access to any (sub)directory is possible and `ls` should list the files. All the filenames and their permissions and metadata (age, size, ownerhip) can be seen.
+After some time, all the files on remote archive server get automatically migrated to the tape. When this happens, entire tree with all the (sub)folders and files can be normally accessed and browsed. `ls` and `cd` commands work just as they work on *regular* filesystem. File names, permissions and metadata (age, size, ownerhip) can be listed.
 
-The difference is that the file content is not directly available anymore - that is, not without calling it back first. If anything is done on the file content, like edit (like compressing, edit with vim/nano) or read (with `less`, `cat` or `grep`) then the command will get stuck in order to retrieve the file from the tapes - which take a lot of time. During this time the command line will be **stuck** unusable.
+The difference is that the file content is not directly available anymore - until it is recalled. If anything is done on the file content (attempt to compress it or modify with vim/nano) or simply read it (with `less`, `cat` or `grep`), will result in the command line **appear stuck**. Actually it is waiting for data to be retrieved. This takes a long time, since the data is copied from tapes to disks.
```

Comment:
```suggestion
The difference is that the file content is not directly available anymore - until it is recalled. Any command that attempts to read the file content may **appear stuck**. Actually it is waiting for data to be retrieved, which may take a long time since the data needs to be copied back from tape to disk.
```